crates/router/src/compatibility/stripe/payment_intents.rs

```diff
@@ -1,6 +1,8 @@
 pub mod types;
 
-use actix_web::{get, post, web, HttpRequest, HttpResponse};
+#[cfg(feature = "olap")]
+use actix_web::get;
+use actix_web::{post, web, HttpRequest, HttpResponse};
```

Comment:
Please ensure that there are no more route handlers that use the `get` and `post` macros, and remove their imports.